### PR TITLE
[main] SELinux policy updates and SELinux size reduction for policy base.

### PR DESCRIPTION
**Pull request by pebenito**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

SELinux updates

###### Change Log  

- Fix policy issues
- Update patches to upstreamed versions
- Split SELinux packages to reduce size on core images.

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline build id: 173249

### CODE REVIEW

File: SPECS/policycoreutils/policycoreutils.spec [hunk 1]
```diff
@@ -274,22 +282,13 @@ The policycoreutils-restorecond package contains the restorecond service.
 %{_mandir}/ru/man8/sepolicy-transition.8*
 %{_mandir}/ru/man8/sepolicy.8*
 
-%files
-%license policycoreutils/COPYING
+%files min
 %{_sbindir}/restorecon
-%{_sbindir}/restorecon_xattr
-%{_sbindir}/fixfiles
```
Comment (MaxBrodeurUrbas) [inline SPECS/policycoreutils/policycoreutils.spec:281]: Is fixfiles not a core functionality? From what I understand it's running restorecon under the hood but I thought it was needed to label system on boot.

File: SPECS/policycoreutils/policycoreutils.spec [hunk 2]
```diff
@@ -351,6 +361,9 @@ The policycoreutils-restorecond package contains the restorecond service.
 %systemd_postun_with_restart restorecond.service
 
 %changelog
+* Tue Mar 08 2022 Chris PeBenito <chpebeni@microsoft.com> - 3.2-1
```
Comment (MaxBrodeurUrbas) [inline SPECS/policycoreutils/policycoreutils.spec:364]: Nit: release number here needs fix

**Review by MaxBrodeurUrbas — APPROVED** [review]
Looks good to me, two small comments.

File: SPECS/policycoreutils/policycoreutils.spec (hunk 1 above)
Comment (pebenito) [reply SPECS/policycoreutils/policycoreutils.spec:281]: No, it's labeled by the build process (images) or install process (iso).

Comment (pebenito) [reply SPECS/policycoreutils/policycoreutils.spec:281]: https://github.com/microsoft/CBL-Mariner/blob/1.0/toolkit/tools/imagegen/installutils/installutils.go#L1519

**Pull request merged**